packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -39,13 +40,14 @@ interface AuthDefaultProps extends AuthFormProps {
   isLoading?: boolean;
   isReady: boolean;
   loginButton?: string;
+  initialEmail?: string;
 }
 
 const AuthDefault = ({
   loginHint,
   onSignup,
   onProviderClick,
-  onForgotPassword,
+  onForgotPassword: handleForgotPassword,
```

Comment:
Why do we need to rename it here? can we not use as is?

Reply:
good catch 👍  leftover code from previous attempts, removed